docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -115,16 +114,16 @@ Use the {product-long-api-designer} web console to create an OpenAPI or AsyncAPI
 
 ifndef::qs[]
 .Prerequisites
-* You're logged in to the {product-api-designer} web console at {service-url-api-designer}[^].
+* You're logged in to the {product-long-api-designer} web console at {service-url-api-designer}[^].
 endif::[]
 
 .Procedure
-. In the {service-url-api-designer}[{product-api-designer} web console], click *Create a schema or API design*.
+. In the {service-url-api-designer}[{product-long-api-designer} web console^], click *Create a schema or API design*.
 . Complete the form to provide the details of the new design:
 .. *Name*: Enter a unique design name, such as `my-api-design`.
```

Comment:
For guides such as _Getting started with RHOSAK_, we've tended to use bullet lists to describe sets of properties that a user must specify. For example, see https://github.com/redhat-developer/app-services-guides/edit/main/docs/kafka/getting-started-kafka/README.adoc?plain=1#L129

Reply:
Done